drivers/core_demographics_report/app/models/core_demographics_report/report_concern.rb

```diff
@@ -78,6 +78,13 @@ def can_see_client_details?(user)
       user.can_access_some_version_of_clients? && ! filter.mask_small_populations
     end
 
+    def can_view_client_disability?(user, disability_id)
+      return user.can_view_hiv_status? if disability_id == 8 # 'HIV/AIDS'
+      return user.can_view_dmh_status? if disability_id == 9 # 'Mental health disorder'
+
+      true
+    end
```

Comment:
This is nice!  I wonder if we should move it somewhere more centralized, but here is good for now.  Thanks!